x/programs/rust/sdk-macros/src/lib.rs

```diff
@@ -257,7 +257,7 @@ pub fn state_keys(_attr: TokenStream, item: TokenStream) -> TokenStream {
 
     // add default attributes
     item_enum.attrs.push(parse_quote! {
-         #[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
```

Comment:
a tab was off 😄

Reply:
It's fine, it's only catching the eye when writing another aligned stuff just below